sql_connectors/connections.py

```diff
@@ -0,0 +1,33 @@
+def setup():
+    import os
+
+    def import_class(cl):
+        last_dot = cl.rfind(".")
+        class_name = cl[last_dot + 1 : len(cl)]
+        module_name = cl[0:last_dot]
+        try:
+            module = __import__(module_name, globals(), locals(), [class_name])
+        except ImportError:
+            raise Exception("Couldn't find module: {}".format(module_name))
```

Comment:
This needs better handling

Reply:
Agree that this is kind of hacky, but seems like it will do what it needs to do. +1 for taking it out of the top level namespace.

Maybe a _little bit_ cleaner of a method would be something like this 
 
```
import os

DEFAULT_PATH_OR_URI = os.environ.get(
        "SQL_CONNECTORS_PATH_OR_URI", "~/.config/sql_connectors"
    )

 def import_class(cl):
        last_dot = cl.rfind(".")
        class_name = cl[last_dot + 1 : len(cl)]
        module_name = cl[0:last_dot]
        try:
            module = __import__(module_name, globals(), locals(), [class_name])
        except ImportError:
            raise Exception("Couldn't find module: {}".format(module_name))

        try:
            return getattr(module, class_name)
        except AttributeError:
            raise Exception("Couldn't find class: {}".format(class_name))

    DEFAULT_STORAGE = os.environ.get(
        "SQL_CONNECTORS_STORAGE", "sql_connectors.storage.LocalStorage"
    )

default_backend = import_class(DEFAULT_STORAGE)
default_connections = default_backend(DEFAULT_PATH_OR_URI)
```

and just importing `default_connections` in the top level `__init__.py`. The reason I like this is that it would be nice to still have access to the global `DEFAULT` variables (for troubleshooting purposes of course). I think the import for a user would still pretty much be the same (`from sql_connectors import default_connections` vs. `from sql_connectors import connections`)

Also might be useful to keep the method in the namespace for programmatically generating configs from metadata if there are different workflows that require different configs (i.e some configs we want to pull from redis and some are local). This would make it easy to parse a `json` or similar file to pipe the appropriate config type to a job

Reply:
Ok - I see the benefit of keeping the import structure the same as a standard module  ; wasn't thinking along those lines.

I was thinking keeping the `import_class` function could be useful for more complicated workflows that requires easily switching between config types, but it's a pretty edge case and I'm not super attached to it.